scripts/install-windows-prereqs.ps1

```diff
@@ -427,6 +427,30 @@ function Install-PSW {
     } -RetryMessage "Failed to start AESMService. Retrying"
 }
 
+function Update-PSW {
+    $tempInstallDir = "$PACKAGES_DIRECTORY\Intel_SGX_PSW"
+    if(Test-Path $tempInstallDir) {
+        Remove-Item -Recurse -Force $tempInstallDir
+    }
+    Install-ZipTool -ZipPath $PACKAGES["psw"]["local_file"] `
+                    -InstallDirectory $tempInstallDir
+
+    $signedZip = Get-Item "$tempInstallDir\Intel*SGX*\PSW_INF_RS3_and_above\component\Signed_*.zip"
+    $infDir = "$tempInstallDir\PSW_INF"
+    Install-ZipTool -ZipPath $signedZip `
+                    -InstallDirectory $infDir
+
+    $infHash = "48e7c1e9-6de8-46ee-8ff9-46aa7b7ee5b9"
```

Comment:
Would it be better to define this as part of `Param`, which is easier to track? It seems like it requires manually update every time  Intel updates the package.